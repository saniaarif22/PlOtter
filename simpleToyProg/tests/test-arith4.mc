/* Test side-effect sequence in a series of statement */

int g;

main() {
  int l;
  l = 1;
  print(l);
  g = 3;
  print(g);
  l = 5;
  print(l+100);
  g = 7;
  print(g+100);
}