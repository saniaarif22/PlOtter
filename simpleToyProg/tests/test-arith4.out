1
3
105
107
